include/z64actor.h

```diff
@@ -1183,4 +1183,98 @@ typedef enum {
+    /* 0x12 */ TATL_HINT_ID_KEESE,
+    /* 0x13 */ TATL_HINT_ID_ARMOS,
+    /* 0x14 */ TATL_HINT_ID_EENO,
+    /* 0x15 */ TATL_HINT_ID_DRAGONFLY,
+    /* 0x16 */ TATL_HINT_ID_IGOS_DU_IKANA,
+    /* 0x17 */ TATL_HINT_ID_GARO,
+    /* 0x18 */ TATL_HINT_ID_GARO_MASTER,
+    /* 0x19 */ TATL_HINT_ID_WART,
+    /* 0x1A */ TATL_HINT_ID_GOMESS,
+    /* 0x1B */ TATL_HINT_ID_GOHT,
+    /* 0x1C */ TATL_HINT_ID_BLUE_BUBBLE,
+    /* 0x1D */ TATL_HINT_ID_LACKEYS_DARK_ROOM,
+    /* 0x1E */ TATL_HINT_ID_LACKEYS_LIGHT_ROOM,
+    /* 0x1F */ TATL_HINT_ID_SKULLWALLTULA,
+    /* 0x20 */ TATL_HINT_ID_GOLD_SKULLTULA, // Only in JP Ver.
+    /* 0x21 */ TATL_HINT_ID_CAPTAIN_KEETA,
+    /* 0x22 */ TATL_HINT_ID_YELLOW_CHUCHU,
+    /* 0x23 */ TATL_HINT_ID_TAKKURI,
+    /* 0x24 */ TATL_HINT_ID_RED_BUBBLE,
+    /* 0x25 */ TATL_HINT_ID_HIPLOOP,
+    /* 0x26 */ TATL_HINT_ID_MASKED_HIPLOOP,
+    /* 0x27 */ TATL_HINT_ID_SHELLBLADE,
+    /* 0x28 */ TATL_HINT_ID_GYORG, // "if you go near, you'll be eaten!"
+    /* 0x29 */ TATL_HINT_ID_GYORG_STUNNED, // "Jump in and attack it!"
+    /* 0x2A */ TATL_HINT_ID_REDEAD,
+    /* 0x2C */ TATL_HINT_ID_BLACK_BOE = 0x2C,
+    /* 0x2D */ TATL_HINT_ID_GIBDO,
+    /* 0x2E */ TATL_HINT_ID_TWINMOLD,
+    /* 0x2F */ TATL_HINT_ID_WEARING_GIANTS_MASK, // Twinmold, JP Ver. only
+    /* 0x30 */ TATL_HINT_ID_WALLMASTER,
+    /* 0x31 */ TATL_HINT_ID_FLOORMASTER,
+    /* 0x32 */ TATL_HINT_ID_MAJORAS_MASK,
+    /* 0x33 */ TATL_HINT_ID_MAJORAS_INCARNATION,
+    /* 0x34 */ TATL_HINT_ID_MAJORAS_WRATH,
+    /* 0x35 */ TATL_HINT_ID_IRON_KNUCKLE,
+    /* 0x37 */ TATL_HINT_ID_LIKE_LIKE = 0x37,
+    /* 0x39 */ TATL_HINT_ID_BEAMOS = 0x39,
+    /* 0x39 */ TATL_HINT_ID_3A, // Empty
+    /* 0x3B */ TATL_HINT_ID_FREEZARD = 0x3B,
+    /* 0x3C */ TATL_HINT_ID_WHITE_BOE,
+    /* 0x42 */ TATL_HINT_ID_OCTOROK = 0x42,
+    /* 0x44 */ TATL_HINT_ID_POE = 0x44,
+    /* 0x45 */ TATL_HINT_ID_GEKKO_SNAPPER,
+    /* 0x46 */ TATL_HINT_ID_BLUE_TEKTITE,
+    /* 0x47 */ TATL_HINT_ID_LEEVER,
+    /* 0x48 */ TATL_HINT_ID_PEAHAT,
+    /* 0x49 */ TATL_HINT_ID_PEAHAT_LARVA,
+    /* 0x4A */ TATL_HINT_ID_EYEGORE,
+    /* 0x4B */ TATL_HINT_ID_WIZROBE,
+    /* 0x4C */ TATL_HINT_ID_WOLFOS,
+    /* 0x4D */ TATL_HINT_ID_MAD_SCRUB,
+    /* 0x50 */ TATL_HINT_ID_MEG = 0x50,
+    /* 0x51 */ TATL_HINT_ID_JO,
+    /* 0x52 */ TATL_HINT_ID_BETH,
+    /* 0x53 */ TATL_HINT_ID_AMY,
+    /* 0x54 */ TATL_HINT_ID_PIRATE,
+    /* 0x55 */ TATL_HINT_ID_STALCHILD,
+    /* 0x56 */ TATL_HINT_ID_ICE_KEESE,
+    /* 0x57 */ TATL_HINT_ID_WHITE_WOLFOS,
+    /* 0x58 */ TATL_HINT_ID_GUAY,
+    /* 0x59 */ TATL_HINT_ID_BIG_OCTO,
+    /* 0x5A */ TATL_HINT_ID_BIG_POE,
+    /* 0x5B */ TATL_HINT_ID_SKULLFISH,
+    /* 0x5C */ TATL_HINT_ID_DESBREKO,
+    /* 0x5D */ TATL_HINT_ID_GREEN_CHUCHU,
+    /* 0x5E */ TATL_HINT_ID_ODOLWA_IDLE,
+    /* 0x5F */ TATL_HINT_ID_GEKKO_GIANT_SLIME,
+    /* 0x60 */ TATL_HINT_ID_BAD_BAT,
+    /* 0x61 */ TATL_HINT_ID_REAL_BOMBCHU,
+    /* 0x62 */ TATL_HINT_ID_ODOLWA_STUNNED,
```

Comment:
I'm still not sure on the trigger conditions for this one, but it's definitely not being stunned; I've seen it use this one while he's not stunned, and here's a screenshot of 0x5E while he's stunned:
![NKZE01_2022-07-30_04-07-12](https://user-images.githubusercontent.com/12245827/181908190-e5ca4704-6341-4b18-a8c9-a39b35d23cbf.png)

I would just call these `ODOLWA_1` and `ODOLWA_2` for now; Elliptic can give these better names when he decomps Odolwa. You can add a short summary to them like you did with the Gyorg ones.